evalutils/templates/processor/{{ cookiecutter.package_name }}/requirements.txt

```diff
@@ -0,0 +1,7 @@
+{% if cookiecutter.dev_build|int %}
+git+git://github.com/comic/evalutils@dev
```

Comment:
Do we really need that on a dev build if it has already been installed previously?

Reply:
I overlooked this statement. the second statement can be removed now, same for evalutils/templates/evaluator/{{ cookiecutter.package_name }}/requirements.txt